client/my-sites/stats/stats-site-overview/index.jsx

```diff
@@ -81,3 +76,20 @@ export default React.createClass( {
 		);
 	}
 } );
+
+export default connect( ( state, ownProps ) => {
+	const { siteId, date, period, siteSlug } = ownProps;
+	const query = {
+		date,
+		period
+	};
+	// It seems not all sites are in the sites/items subtree consistently
```

Comment:
Thanks for the clarification - I didn't dig into the why of that yet, so much appreciated on the explanation